### ISSUE TITLE
Put and Yank Highlighting not working on Diff

### ISSUE THREAD

**Issue by marcoSven** [gbprod/yanky.nvim#115]
Is this something in my config or a known issue that highlighting is not able to overrule diff background color?

Yank highlight background should be yellow in my case

```lua
DiffChange = {bg = "#1E2C32"},
YankyYanked = {fg = c.background, bg = c.yellow},
```

https://github.com/gbprod/yanky.nvim/assets/3860298/5c7ebafd-1576-4d8d-9c70-8dd2eabb4c53

**Comment by gbprod** [comment]
By default, `YankyYanked` highlight group is linked to `Search`. Maysbe there is someting in your config or your theme that change this ?

Can you check the result of `:hi YankyYanked` and `:hi Search` ?

**Comment by marcoSven** [comment]
I defined hl `YankyYanked` in my theme but I removed it to test if Search hl works. 

> Can you check the result of :hi YankyYanked and :hi Search

After removing `YankyYanked` from my theme

`YankyYanked   xxx cterm= gui= links to Search`
`Search        xxx guifg=#292828 guibg=#d8a657`


Search highlighting works, found text unlike yanked text is highlighted correctly. Maybe an issue with `vim.highlight.on_yank`?
